test/OperationTest.cpp

```diff
@@ -415,9 +418,7 @@ TEST(Operation, verifyRuntimeInformationIsUpdatedForLazyOperations) {
 // _____________________________________________________________________________
 TEST(Operation, ensureFailedStatusIsSetWhenGeneratorThrowsException) {
   bool signaledUpdate = false;
-  Index index = makeTestIndex(
-      "ensureFailedStatusIsSetWhenGeneratorThrowsException", std::nullopt, true,
-      true, true, ad_utility::MemorySize::bytes(16), false);
```

Comment:
Is it intentional that all of these got changed from 16 to 32?

Reply:
In fact it doesn't matter for those tests, so I don't care
(the tests don't check for individual block properties).